tests/fullydistributed_grids/copy_distributed_tria_01.with_p4est=true.with_metis=true.mpirun=7.output

```diff
@@ -0,0 +1,13 @@
+
```

Comment:
why do you need metis?

Reply:
I just noticed that this had a reason: I was using `GridTools::partition_triangulation` with metis as partitioner. I have replaced this function call with `GridTools::partition_triangulation_zorder`, which neither depends on metis nor on zoltan!